tests/pragmas/tbitsize.nim

```diff
@@ -0,0 +1,18 @@
+type
+  bits* = object
+    flag* {.bitsize: 1.}: cuint
+    opts* {.bitsize: 4.}: cint
+
+var
+  b: bits
+
```

Comment:
On my machine `sizeof(bits) == 4`, is it safe to assume this will be consistent across platforms?

The `assert (b.flag = 2) == 0` below should be a pretty reliable test since it ensures the field is only one bit long.